src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -1311,6 +1311,13 @@ public FFIMethodAttributes(MethodInfo method, Dictionary<MethodInfo, Attribute[]
                     var visibleInLibraryAttr = attr as IsVisibleInDynamoLibraryAttribute;
                     HiddenInLibrary = (visibleInLibraryAttr.Visible == false);
                 }
+                else if (attr is IsObsoleteAttribute)
+                {
+                    HiddenInLibrary = true;
+                    ObsoleteMessage = (attr as IsObsoleteAttribute).Message;
+                    if (string.IsNullOrEmpty(ObsoleteMessage))
+                        ObsoleteMessage = "Obsolete";
+                }
                 else if (attr is ObsoleteAttribute)
```

Comment:
ObsoleetAttribute is system attribute `Obsolete`, which is a sealed class, so we can't have localized message for it.